src/cmdline.ggo

```diff
@@ -1,6 +1,8 @@
 package "pflask"
-version "0.2"
+version "0.3"
 
+option "caps"  b "Specify comma-separated capabilities to add ('+' prefix) or drop ('-' prefix)"
+       string default="+all" optional multiple
```

Comment:
Only drawback I see with doing it this way is that if there are multiple ``all`` specified, and they get de-duplicated, then you get an error about ``all`` not being the first argument. E.g.:
```
# pflask --caps all,-chown,all
[✘] Alias 'all' is valid only as first capability
[✘] Child failed with code '1'
```

It's not documented and might leave the user surprised, but I think it's okay to leave it like this.